libs/feature-settings/src/lib/feature-settings.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Remove this empty spec file or add some tests

Reply:
Good point